comments/functions.js

```diff
@@ -0,0 +1,73 @@
+$(document).ready(function(){
+
+   function renderTemplate (templateId, variables) {
+      return Handlebars.compile($(templateId).html())(variables)
+   }
+
+   $(".comments__list").sortable({
+      revert: true
+    });
+
+   $(".comments").on("click", "input[name=hide-form]", function(){
+      $("form[name=update], form[name=answer]").hide();
```

Comment:
You need to use constants for thigs like this 'form[name=answer]'. Please create constant FROM_ANSWER = 'form[name=answer]' and use it in your code.